Sample error log file:

[28/Jun/2021:18:12:08.539] ERROR app.logger:wrapper: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Server': 'nginx/1.10.3 (Ubuntu)', 'Date': 'Mon, 28 Jun 2021 12:42:08 GMT', 'Content-Type': 'application/json', 'Content-Length': '222', 'Connection': 'keep-alive', 'Cache-Control': 'no-cache, private', 'X-Kubernetes-Pf-Flowschema-Uid': '00dde116-c112-4c2f-b28e-395bfac35cd7', 'X-Kubernetes-Pf-Prioritylevel-Uid': '75420dee-645c-4488-a3c3-a01ebdf2f742'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"deployments.apps \"deployment\" not found","reason":"NotFound","details":{"name":"deployment","group":"apps","kind":"deployments"},"code":404}


Traceback (most recent call last):
  File "C:\Path\coxswain\coxswain\kube.py", line 17, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Path\coxswain\coxswain\kube.py", line 187, in updateDeploymentImage
    response = self.k8s_apps_v1.read_namespaced_deployment(name, namespace)
  File "C:\Path_to_packages\kubernetes\client\api\apps_v1_api.py", line 6317, in read_namespaced_deployment
    return self.read_namespaced_deployment_with_http_info(name, namespace, **kwargs)  # noqa: E501
  File "C:\Path_to_packages\kubernetes\client\api\apps_v1_api.py", line 6412, in read_namespaced_deployment_with_http_info
    return self.api_client.call_api(
  File "C:\Path_to_packages\kubernetes\client\api_client.py", line 348, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Path_to_packages\kubernetes\client\api_client.py", line 180, in __call_api
    response_data = self.request(
  File "C:\Path_to_packages\kubernetes\client\api_client.py", line 373, in request
    return self.rest_client.GET(url,
  File "C:\Path_to_packages\kubernetes\client\rest.py", line 239, in GET
    return self.request("GET", url,
  File "C:\Path_to_packages\kubernetes\client\rest.py", line 233, in request
    raise ApiException(http_resp=r)
kubernetes.client.exceptions.ApiException: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Server': 'nginx/1.10.3 (Ubuntu)', 'Date': 'Mon, 28 Jun 2021 12:42:08 GMT', 'Content-Type': 'application/json', 'Content-Length': '222', 'Connection': 'keep-alive', 'Cache-Control': 'no-cache, private', 'X-Kubernetes-Pf-Flowschema-Uid': '00dde116-c112-4c2f-b28e-395bfac35cd7', 'X-Kubernetes-Pf-Prioritylevel-Uid': '75420dee-645c-4488-a3c3-a01ebdf2f742'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"deployments.apps \"deployment\" not found","reason":"NotFound","details":{"name":"deployment","group":"apps","kind":"deployments"},"code":404}
